### DISCUSSION TITLE
Base Permission on a repository

### DISCUSSION THREAD

**Discussion by meiswjn** [community/community#22063] [New to GitHub]
<p>Hey :slight_smile:</p>
<p>I am trying to setup an organization, in which every member has <em>read</em> access to internal repositories and no access to private repositories.</p>
<p>The docs say, internal repositories can be read by everyone - that is correct, but people cannot even star the repository, nor can they open pull-requests. Internal projects should have the triage-permission level, or atleast we should have the option to grant this permission level to everyone.</p>
<p>Scenario:<br>
I have an internal onboarding repository in which I want people to experience forks and pull requests. Sadly, I cannot grant everyone triage permissions only for that repository. It is not ‘public’ on an internal basis. I have to invite every user or team one by one as triage - thats horrible. And if I make the base role (which also doesnt support triage btw) read-only, private repositorys become the same as internal projects - that doesn’t make any sense either.</p>
<p>How do I make all of our users automatically triages in that onboarding repo?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/meiswjn">@meiswjn</a>, thank you for being here! On GitHub enterprise you can try <a href="https://help.github.com/enterprise/admin/guides/installation/enabling-private-mode/">enabling private mode</a> so that only authenticated users can access private repositories.</p>

**Comment by meiswjn** [comment, marked as answer]
<p>Thanks for the reply, <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a>.<br>
Private mode is for GH Servers only, not the cloud.<br>
And as much as I appreciate a reply from GitHub, private mode is not enabled to my problem at all.</p>
<p>The Roadmap teasers a solution for my issue by Q4.<br>
Thanks, nevertheless.</p>

**Comment by JonathanJohnson023** [comment]
What about setting tirage as the base role for all private repositories @AndreaGriffiths11
